src/about/components/item-responsive.jsx

```diff
@@ -0,0 +1,9 @@
+import { Grid } from '@material-ui/core'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+export const ItemResponsive = ({ children }) => <Grid item xs={6} sm={6} md={6}>
```

Comment:
By reading this component's name but without looking into it, I get the following info:
* Since I know how Material UI works, I can tell this is a Grid Item
* I know it is responsive

The problem is I have no idea in what way it is responsive. Is it fullscreen? Halfscreen? Only visible on mobile?
So maybe we should rename it to something like `<HalfRowItem/>` or something to that effect.